test/docs/OpenJ9TestUserGuide.md

```diff
@@ -49,6 +49,7 @@ tools should be installed on your test machine to run tests.
     SPEC=[linux_x86-64|linux_x86-64_cmprssptrs|...] (platform on which to test)
     JAVA_VERSION=[SE80|SE90|SE100|SE110|Panama|Valhalla] (SE90 default value)
     JAVA_IMPL=[openj9|hotspot|sap] (openj9 default value)
+    BUILD_LIST=<comma separated projects to be compiled and executed> (default to all projects)
```

Comment:
I just realized that we have this section called "required environment variables" which someone might read as you need to export them, but since many have a default value, we could just change it to "environment variables"